README.md

```diff
@@ -101,11 +101,9 @@ curl base_url/community/create/ \
 
 **Request :**
 ```
-curl base_url/community/add_user/ \
+curl base_url/community/<sample_user_name> \
```

Comment:
Is this the endpoint for adding a user to a community? If that is the case, what do you guys @roy @omkarjc27 think about the endpoint being something like `/communities/:community_id/:user_id`?

Reply:
That would surely make the method more readable.

But community name in the URL is not required as we will be able to identify the community based on the API KEY of that community.